apps/openmw/mwmechanics/aiwander.cpp

```diff
@@ -201,7 +201,18 @@ namespace MWMechanics
             stopWalking(actor, storage);
             currentCell = actor.getCell();
             storage.mPopulateAvailableNodes = true;
-            mStoredInitialActorPosition = false;
+        }
+
+        // Here we should reset an initial position, if a current cell was REALLY changed
+        // We do not store AiStorage in a savegame, so cellChange is not help us in this case
+        // TODO: find a more simple and fast solution, or do not store the mInitialActorPosition at all
+        if (mStoredInitialActorPosition)
```

Comment:
Is this still needed?

Reply:
> Is this still needed?

For now the initial position is still used in the wanderNearStart() function.